src/ts/components/navigation/side-bar.tsx

```diff
@@ -23,6 +23,12 @@ export interface SideBarProps extends HTMLProps<HTMLElement>, ComponentProps {
   onClickOutside(event: React.MouseEvent<HTMLDivElement>): void;
 }
 
+const TIMEOUT = {
```

Comment:
These timeouts are duplicated a lot?

Reply:
Yeah, I had a feeling whoever reviewed would comment on that. XD

The fact that they happen to be the same is just coincidence. I've chosen to keep them as separate constants to avoid people updating one transition and breaking the others.